core/snippets/snippets/commentBlock.snippet

```diff
@@ -0,0 +1,21 @@
+name: commentBlock
+phrase: block comment
+insertionScope: statement
+
+$0.insertionFormatter: SENTENCE
```

Comment:
I believe this needs to be lower case sentence to match the tip?
`2023-10-30 23:17:13.940 ERROR cb error topic="phrase" cb=SpeechSystem.engine_event
   17:             talon\lib\_thread.py:19  * # cron thread
   16:                     threading.py:995 * 
   15:                     threading.py:1038* 
   14:                     threading.py:975 * 
   13:                    talon\cron.py:156 | 
   12:                     talon\vad.py:23  | 
   11:                     talon\vad.py:129 | 
   10: talon\scripting\speech_system.py:370 | 
    9:             talon\engines\w2l.py:757 | 
    8:      talon\scripting\dispatch.py:134 | # 'phrase' main:EngineProxy._redispatch()
    7: talon\scripting\speech_system.py:67  | 
    6: -------------------------------------# [stack splice]
    5:      talon\scripting\dispatch.py:134 | # 'phrase' main:SpeechSystem.engine_event()
    4: talon\scripting\speech_system.py:487 | 
    3:      talon\scripting\executor.py:111 | 
    2:  talon\scripting\talon_script.py:707 | 
    1:  talon\scripting\talon_script.py:610 | 
talon.scripting.talon_script.TalonScriptError: 
 in script at c:\users\jeffrey.knaus\appdata\roaming\talon\user\knausj_talon\core\snippets\snippets.talon:4:
 > user.insert_snippet_by_name_with_phrase(snippet_insert_with_phrase, text)
KeyError: 'SENTENCE'`

Reply:
Oh my formatters are not called the same as community. I will have to do a find and replace.

Reply:
Done!